src/main/plugins/org.dita.pdf2/cfg/common/vars/ar.xml

```diff
@@ -328,16 +328,46 @@ See the accompanying license.txt file for applicable licenses.
 
     <!-- The header that appears on the even-numbered glossary pages. -->
     <variable id="Glossary even header"><param ref-name="pagenum"/> | <param ref-name="heading"/> | <param ref-name="prodname"/></variable>
-     
+
      <!-- The heading string to put at the top of the Glossary -->
      <variable id="Glossary">قاموس المصطلحات</variable>
-     
+
      <!-- The heading string to put at the top of the List of Tables -->
      <variable id="List of Tables">الجداول</variable>
-     
+
      <!-- The heading string to put at the top of the List of Figures -->
      <variable id="List of Figures">الأشكال</variable>
 
+     <!-- The heading string to put at the top of the List of Changes -->
+     <variable id="List of Changes"></variable>
```

Comment:
@jelovirt - that is what I did for the few items that I didn't have when adding languages last fall. Some of those, I couldn't actually see where the string was used (it was referenced in code but I couldn't find the markup to make it appear).

I think there is an argument either way (leave empty, or leave English). I went with English because that's more likely to be noticed and fixed if the string is ever needed. Empty strings will probably leave messed up output that wouldn't be noticed if doing a quick QA scan over the file.